src/frontend/test_runner/tests/testdata/range_scan.yaml

```diff
@@ -32,22 +32,22 @@
   - create_table_and_mv
   sql: |
     SELECT * FROM orders_count_by_user WHERE user_id = 1/0
-  batch_error: 'Expr error: DivisionByZero'
+  batch_error: 'Expr error: Division by zero'
 - before:
   - create_table_and_mv
   sql: |
     SELECT * FROM orders_count_by_user WHERE user_id = 2147483647 + 1
-  batch_error: 'Expr error: NumericOutOfRange'
+  batch_error: 'Expr error: Out of range'
 - before:
   - create_table_and_mv
   sql: |
     SELECT * FROM orders_count_by_user WHERE user_id = 'a'
-  batch_error: 'Expr error: Cast("str", "i64")'
+  batch_error: 'Expr error: Can''t cast str to i64'
```

Comment:
What's wrong 😄

Reply:
Previously errors were displayed in debug mode.

Reply:
I mean the `Can''t` has an extra `'`...

Reply:
Escape char of yaml?

Reply:
Yes, it is an escape char, because of `'` `'` around `Can't`

Reply:
got it